### PR TITLE
chore: prepare release 2023-08-16

### PR DESCRIPTION
**Pull request by shortcuts**
## Summary

This PR has been created using the `yarn release` script. Once merged, the clients will try to release their new version if their version has changed.

## Version Changes

- javascript: 5.0.0-alpha.78 -> **`prerelease` _(e.g. 5.0.0-alpha.79)_**
- java: 4.0.0-beta.3 -> **`prerelease` _(e.g. 4.0.0-beta.4)_**
- php: 4.0.0-alpha.75 -> **`prerelease` _(e.g. 4.0.0-alpha.76)_**
- go: 4.0.0-alpha.24 -> **`prerelease` _(e.g. 4.0.0-alpha.25)_**
- kotlin: 3.0.0-SNAPSHOT -> **`patch` _(e.g. 3.0.0-SNAPSHOT)_**
- dart: 0.3.0 -> **`patch` _(e.g. 0.3.1)_**

### Skipped Commits

<p>It doesn't mean these commits are being excluded from the release. It means they're not taken into account when the release process figured out the next version number, and updated the changelog.</p>

<details>
  <summary>
    <i>Commits without language scope:</i>
  </summary>

  - chore: prepare release 2023-08-08 (#1888)
</details>

<details>
  <summary>
    <i>Commits with unknown language scope:</i>
  </summary>

  - chore(deps): dependencies 2023-08-14 (#1895)
</details>

### CODE REVIEW

File: clients/algoliasearch-client-dart/packages/client_core/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: algolia_client_core
 description: Algolia Client Core is a Dart package for seamless Algolia API integration, offering HTTP request handling, retry strategy, and robust exception management.
-version: 0.2.0
+version: 0.2.0+4
```
Comment (Fluf22) [inline clients/algoliasearch-client-dart/packages/client_core/pubspec.yaml:3]: Isn't this weird? 🤔
I'm a little lost with the latest changes there was on the versioning of the different clients 😄

Comment (shortcuts) [reply clients/algoliasearch-client-dart/packages/client_core/pubspec.yaml:3]: this is definitely wrong, I forgot about https://github.com/algolia/api-clients-automation/pull/1890 and I think @aallam is off

Comment (shortcuts) [reply clients/algoliasearch-client-dart/packages/client_core/pubspec.yaml:3]: I'll manually bump it for this release I think

Comment (Fluf22) [reply clients/algoliasearch-client-dart/packages/client_core/pubspec.yaml]: AH! Nice, I'm still up to it 😏

Comment (shortcuts) [reply clients/algoliasearch-client-dart/packages/client_core/pubspec.yaml]: Even the changelog wasn't correct 🤔 I wonder which tags it is fetching from

**Pull request merged**